docker/Dockerfile

```
@@ -26,9 +26,9 @@ RUN echo "---- INSTALL RUNTIME PACKAGES ----" && \
     && echo "date.timezone = UTC" >> /etc/php8/php.ini \
     && ln -s /usr/bin/php8 /bin/php
 
-# Add gosu for easy step-down from root
-RUN apk add --no-cache --update --upgrade --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing \
-    gosu
+# Add su-exec for easy step-down from root (gosu deprecated in Alpine 3.15+)
+RUN apk add --no-cache --update --upgrade \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)